packages/redot-parse/index.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line n/file-extension-in-import
+import {parse} from 'ts-graphviz/ast'
+
+/** @type {import('unified').ParserFunction<import('ts-graphviz/ast').DotASTNode>} */
+function parser(document, vfile) {
+  return parse(document, {startRule: 'Dot', filename: vfile.path})
```

Comment:
Q: should there be configuration?
Q: is `DotASTNode` a unist compatible node? Shouldn’t it subclass the same way `types/hast` or so works based on `types/unist`?

Reply:
> Q: should there be configuration?

Potentially? I could do a pass through of the `ts-graphviz` options.

> Q: is DotASTNode a unist compatible node?

It is, it uses type and children the same as unist specifies.

> Shouldn’t it subclass the same way types/hast or so works based on types/unist?

Maybe? What would be the benefit of subclassing here? Strong types on `data`? (I thought that was going away)